### PR TITLE
enhance: Add database param to manual compact api

### PR DESCRIPTION
**Pull request by weiliu1031**
issue: milvus-io/milvus#38086

### CODE REVIEW

File: proto/milvus.proto [hunk 1]
```diff
@@ -1216,6 +1216,7 @@ message ManualCompactionRequest {
   uint64 timetravel = 2;
   bool majorCompaction = 3;
   string collection_name = 4;
+  string db_name = 5;
```
Comment (czs007) [inline proto/milvus.proto:1219]: good job

**Comment by SimFG** [comment]
ahhh

File: proto/milvus.proto (hunk 1 above)
Comment (SimFG) [reply proto/milvus.proto:1219]: ahhh

**Pull request merged**